components/activity_and_insights/insights/card/card.tsx

```diff
@@ -48,6 +51,15 @@ const InsightsCard = (props: Props) => {
                         e.stopPropagation();
                         props.onClick();
                     }}
+                    aria-label={formatMessage(
+                        {
+                            id: 'insights.card.ariaLabel', 
+                            defaultMessage: 'Open modal for {title}'
+                        },
+                        {
+                            title: props.title,
+                        }
+                    )}
                 >
```

Comment:
Maybe add `aria-haspopup={true}`?

Reply:
isnt that attribute used for popover instead of modal?

Reply:
It's used for a modal as well